spec/outputs/graphite_spec.rb

```diff
@@ -1,86 +1,49 @@
 require_relative '../spec_helper'
 
-describe LogStash::Outputs::Graphite do
-
-  let(:port) { 4939 }
-  let(:config) do <<-CONFIG
-     input {
-        generator {
-          message => "foo=fancy bar=42"
-          count => 1
-          type => "generator"
-        }
-      }
-
-      filter {
-        kv { }
-      }
-
-      output {
-        graphite {
-          host => "localhost"
-          port => #{port}
-          metrics => [ "hurray.%{foo}", "%{bar}" ]
-        }
-      }
-  CONFIG
+class LogStash::Outputs::Graphite
```

Comment:
I don't think this pollutes the file, it is an essential part of the setup to this test suite.